PWGCF/DataModel/FemtoDerived.h

```diff
@@ -112,14 +112,24 @@ DECLARE_SOA_DYNAMIC_COLUMN(TPCCrossedRowsOverFindableCls, tpcCrossedRowsOverFind
                            [](uint8_t tpcNClsFindable, uint8_t tpcNClsCrossedRows) -> float {
                              return (float)tpcNClsCrossedRows / (float)tpcNClsFindable;
                            });
+DECLARE_SOA_COLUMN(TPCNSigmaEl, tpcNSigmaEl, float);
+DECLARE_SOA_COLUMN(TPCNSigmaPi, tpcNSigmaPi, float);
+DECLARE_SOA_COLUMN(TPCNSigmaKa, tpcNSigmaKa, float);
+DECLARE_SOA_COLUMN(TPCNSigmaPr, tpcNSigmaPr, float);
+DECLARE_SOA_COLUMN(TPCNSigmaDe, tpcNSigmaDe, float);
+DECLARE_SOA_COLUMN(TOFNSigmaEl, tofNSigmaEl, float);
+DECLARE_SOA_COLUMN(TOFNSigmaPi, tofNSigmaPi, float);
+DECLARE_SOA_COLUMN(TOFNSigmaKa, tofNSigmaKa, float);
+DECLARE_SOA_COLUMN(TOFNSigmaPr, tofNSigmaPr, float);
+DECLARE_SOA_COLUMN(TOFNSigmaDe, tofNSigmaDe, float);
```

Comment:
Please, in the next iteration do comment the column declaration so that it is made available in the documentation